src/utils/formatError.ts

```diff
@@ -0,0 +1,16 @@
+export default (err: any, message?: string): {
+  name?: string;
+  message: string;
+  stack?: string;
+} => {
+  if (err instanceof Error) {
```

Comment:
If you want to check whether something is a built-in `Error` object, this is better:

```suggestion
  if (Object.prototype.toString.call(err) === '[object Error]') {
```

since `instanceof` is not a reliable brand check in JS:

```js
> vm.runInNewContext('new Error("banana")')
Error: banana
    at evalmachine.<anonymous>:1:1
    [… rest of stack …]
> vm.runInNewContext('new Error("banana")') instanceof Error
false
```

or alternatively I’d be more specific and just test whether the actual properties you’re looking for are present:

```suggestion
  if (typeof err?.name === 'string' && typeof err?.message === 'string' && typeof err?.stack === 'string') {  
```

Reply:
nice!! thank you for the suggestion! ❤️